doc/about.rst

```diff
@@ -188,29 +278,43 @@ program.
 
 .. _Vlad Niculae: https://vene.ro/
 
-It also provided funding for sprints and events around scikit-learn. If
-you would like to participate in the next Google Summer of code
-program, please see `this page
-<https://github.com/scikit-learn/scikit-learn/wiki/SummerOfCode>`_.
+...................
 
 The `NeuroDebian <http://neuro.debian.net>`_ project providing `Debian
 <https://www.debian.org/>`_ packaging and contributions is supported by
 `Dr. James V. Haxby <http://haxbylab.dartmouth.edu/>`_ (`Dartmouth
 College <https://pbs.dartmouth.edu/>`_).
 
-The `PSF <https://www.python.org/psf/>`_ helped find and manage funding for our
-2011 Granada sprint. More information can be found `here
-<https://github.com/scikit-learn/scikit-learn/wiki/Past-sprints#granada-19th-21th-dec-2011>`__
+Sprints
+-------
+
+The International 2019 Paris sprint was kindly hosted by `AXA <https://www.axa.fr/>`_.
+Also some participants could attend thanks to the support of the `Alfred P.
+Sloan Foundation <https://sloan.org>`_, the `Python Software
+Foundation <https://www.python.org/psf/>`_ (PSF) and the `DATAIA Institute
+<https://dataia.eu/en>`_.
 
-`tinyclues <https://www.tinyclues.com/>`_ funded the 2011 international Granada
-sprint.
+.....................
 
+The 2013 International Paris Sprint was made possible thanks to the support of
+`Télécom Paristech <https://www.telecom-paristech.fr/>`_, `tinyclues
+<https://www.tinyclues.com/>`_, the `French Python Association
+<https://www.afpy.org/>`_ and the `Fonds de la Recherche Scientifique
+<https://www.frs-fnrs.be/-fnrs>`_.
+
+..............
+
+The 2011 International Granada sprint was made possible thanks to the support
+of the `PSF <https://www.python.org/psf/>`_ and `tinyclues
+<https://www.tinyclues.com/>`_.
 
 Donating to the project
```

Comment:
See also the discussion about NumFocus here #6603
@GaelVaroquaux and Co. (I don't know exactly who I need to ping...) is that still relevant? Who is taking care of that?
Thanks!